claim_ai_quality/apps.py

```diff
@@ -13,14 +13,13 @@
 class ClaimAiQualityConfig(AppConfig):
     name = MODULE_NAME
 
-    claim_ai_url = ""
-    test = ""
+    claim_ai_url = "ws://localhost:8000/claim_ai/ws/Claim/process/"
     event_based_activation = False
     bundle_size = 100
     zip_bundle = False
 
     def _configure_perms(self, cfg):
-        ClaimAiQualityConfig.claim_ai_url = cfg.get("claim_ai_url", "")
+        ClaimAiQualityConfig.claim_ai_url = cfg.get("claim_ai_url", "ws://localhost:8000/claim_ai/ws/Claim/process/")
```

Comment:
Can't we replace the string with the variable claim_ai_url as it has same value? Requires updates in only one place.